packages/integration-tests/test/000_withdrawals.spec.ts

```diff
@@ -0,0 +1,212 @@
+// Named 000 in order to run first since the output submitter
+// can fall behind.
+
+/* Imports: External */
+import { BigNumber, constants, Contract, ContractReceipt, utils, Wallet } from 'ethers'
+import { awaitCondition } from '@eth-optimism/core-utils'
+import * as rlp from 'rlp'
+import { Block } from '@ethersproject/abstract-provider'
+
+/* Imports: Internal */
+import { WITHDRAWER_ADDR } from './shared/constants'
+import env from './shared/env'
+import { expect } from './shared/setup'
+import winston from 'winston'
+
+const withdrawerArtifact = require('../../contracts/artifacts/contracts/L2/Withdrawer.sol/Withdrawer.json')
+const l2OOracleArtifact = require('../../contracts/artifacts/contracts/L1/L2OutputOracle.sol/L2OutputOracle.json')
+const counterArtifact = require('../artifacts/Counter.sol/Counter.json')
+const multiDepositorArtifact = require('../artifacts/MultiDepositor.sol/MultiDepositor.json')
+
+describe('Withdrawals', () => {
+  let logger: winston.Logger
+  let portal: Contract
+  let withdrawer: Contract
+
+  let recipient: Wallet
+
+  before(async () => {
+    logger = env.logger
+    portal = env.optimismPortal
+
+    withdrawer = new Contract(
+      WITHDRAWER_ADDR,
+      withdrawerArtifact.abi,
+    )
+  })
+
+  describe('simple withdrawals', () => {
+    let nonce: BigNumber
+    let burnBlock: Block
+    let withdrawalHash: string
+    const value = utils.parseEther('1')
+    const gasLimit = 3000000
+
+    before(async function () {
+      this.timeout(60_000)
+      recipient = Wallet.createRandom().connect(env.l2Provider)
+      withdrawer = withdrawer.connect(recipient)
+
+      logger.info('Generated new wallet', {
+        recipient: recipient.address,
+      })
+      logger.info('Depositing to new address on L2')
+      let tx = await portal.connect(env.l1Wallet).depositTransaction(
```

Comment:
You should be able to send value directly to the contract without needing to call `depositTransaction`, could remove some lines of code

Reply:
I kind of like being explicit about calling the contract rather than just sending value - it makes it clear that we're calling the portal, and what parameters we're setting.